components/Image/Image.tsx

```diff
@@ -47,7 +46,8 @@ export default function Image ( props: Props ) {
 	return (
 		<NextImage
 			loader={loader}
-			{...imageProps}
```

Comment:
@ingeniumed Minor, but I noticed that we use the `NextImage`-specific property `fill` here on line 45:

https://github.com/Automattic/vip-go-nextjs-skeleton/blob/6620e2ae053c98fbd5bbb4b3be62a98157dbc737/pages/media/index.tsx#L42-L47

which will get dropped by the changes. Is there a way to change things so `Image` still properly passes inner properties to `NextImage`?